contents/docs/contribute/contribute-to-website.md

```diff
@@ -9,7 +9,7 @@ You can contribute to the PostHog documentation, handbook, and blog in two ways:
 1. You can create a Pull Request in GitHub for any page that has an **Edit this page** link on it. In this situation you must edit the page using the GitHub web editor interface. This method is suitable for text-only edits and basic file manipulation, such as renaming.
 2. You can run the posthog.com website locally and make changes there by creating a branch of the master codebase, committing changes to that branch and raising a Pull Request to merge those changes. This is the recommended method as it allows you to quickly preview your changes, as well as perform more complex changes easily.
 
-Below, we'll explain how to set up option two. Most PostHog employees use Mac laptops in their day-to-day work, so this guide focuses on making changes via a Mac, rather than a Windows computer.
+Below, we'll explain how to set up option two.
```

Comment:
This is a guide for everyone. Calling out employees doesn't seem right here.

I believe the guides here will work on Mac, Linux & Windows.

Reply:
Happy if you want to drop this, but surely this isn't going to work on PC because we're referencing things (such as Terminal) which aren't on PC (unless you're talking about Command Prompt). 

You're right this is probably too dumped down though.